drivers/scheduler/k8s/k8s.go

```diff
@@ -534,6 +608,12 @@ func (k *K8s) Schedule(instanceID string, options scheduler.ScheduleOptions) ([]
 			return nil, err
 		}
 
+		helmSpecObjects, err := k.HelmSchedule(app, appNamespace, options)
+		if err != nil {
+			return nil, err
+		}
+
+		specObjects = append(specObjects, helmSpecObjects...)
```

Comment:
I think this will fail for existing tests as thy don't use helm. Try run SetupTearDown test on your branch to see if it works.

Reply:
existing tests won't use helm, so they should work fine. that's being said if the app doesn't have a helm spec, it won't be scheduled through helm.